### PR TITLE
Mindustry category fix

### LINKED ISSUE #310229: Mindustry appears under Lost & Found category
### Describe the bug
Mindustry is placed under `Lost & Found` in the KDE application launcher.

### Steps To Reproduce
1. Install KDE
2. Install mindustry

### Expected behavior
Mindustry should be under the `Games` category.

### Screenshots
image

### Notify maintainers
@chkno @fgaz @thekostins

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.1.86, NixOS, 23.11 (Tapir), 23.11.20240417.e402c3e`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.18.1`
 - channels(root): `"nixos, nixpkgs"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

### PR DESCRIPTION
**Pull request by importantblimp**
## Description of changes
Add mindustry to the Games category

Fixes #310229

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by importantblimp** [comment]
I've run the passthru tests (`nix-build --attr pkgs.mindustry.passthru.tests`) but I haven't been able to work out how to build my (flake) NixOS system using my local copy of nixpkgs.

Though the generated .desktop file is
```
[Desktop Entry]
Categories=Game
Exec=mindustry
Icon=mindustry
Name=Mindustry
Type=Application
Version=1.4
```
which is what I expected

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -8659,6 +8659,11 @@
       fingerprint = "F5B2 BE1B 9AAD 98FE 2916  5597 3665 FFF7 9D38 7BAA";
     }];
   };
+  importantblimp = {
+    github = "importantblimp";
+    githubId = 20533356;
+    name = "GY";
+  };
```
Comment (Sigmanificient) [inline maintainers/maintainer-list.nix:8666]: You do not need to add yourself in the maintainer list, if you do not add yourself in to the package `meta.maintainers` list

**Review by Sigmanificient — CHANGES_REQUESTED** [review]

**Comment by Sigmanificient** [comment]
Fixes: #310229

File: maintainers/maintainer-list.nix (hunk 1 above)
Comment (importantblimp) [reply maintainers/maintainer-list.nix]: Good to know, thanks - removed

**Review by fgaz — APPROVED** [review]
Thanks!

**Pull request merged**